app/controllers/application_controller.rb

```diff
@@ -3,6 +3,7 @@
 class ApplicationController < ActionController::API
   before_action :authenticate
   load_and_authorize_resource
+  serialization_scope :current_user
```

Comment:
This is a base controller, adding scope here will impact everything, if you are sure about the impact, please add your justification here.